jenkins/databricks/clusterutils.py

```diff
@@ -0,0 +1,149 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime
+import json
+import time
+import os
+import requests
+import sys
+
+class ClusterUtils(object):
+
+    @staticmethod
+    def generate_create_templ(sshKey, initScriptFile, version):
+        # TODO (in followup) - create directory if not there
```

Comment:
Is there a follow on issue for this?  Or do you plan to tackle it when/if it becomes a problem?

Reply:
this was for enhancing to do dev builds and such. Or if we add more testing envs.

Reply:
I'll just remove it for now